tests/rest/client/v1/utils.py

```diff
@@ -152,7 +161,7 @@ def send_state(self, room_id, event_type, body, tok, expect_code=200, state_key=
             path = path + "?access_token=%s" % tok
 
         request, channel = make_request(
-            self.hs.get_reactor(), "PUT", path, json.dumps(body).encode("utf8")
+            self.hs.get_reactor(), method, path, json.dumps(body).encode("utf8")
```

Comment:
Seems a bit weird to me that we're trying to send a body even when we're doing a GET request.